ign_ros2_control/src/ign_system.cpp

```diff
@@ -589,7 +607,7 @@ hardware_interface::return_type IgnitionSystem::write(
       {
         this->dataPtr->ecm->CreateComponent(
           this->dataPtr->joints_[i].sim_joint,
-          ignition::gazebo::components::JointVelocityCmd({0}));
```

Comment:
I don't believe it makes sense to initialize this to 0 when we have a command available.